### DISCUSSION TITLE
Idea to enable "DO_NOT_TRANSCODE_VIDEO" by default?

### DISCUSSION THREAD

**Discussion by foeyonghai** [mediacms-io/mediacms#1171] [Feature Requests]
Hi @mgogoulos.

I also mentioned it in https://github.com/mediacms-io/mediacms/issues/1098.

Do you have an idea of how important the transcode function is to have it enabled by default and if it still should be enabled by default? It required a lot of CPU power and hard disk space.

The question is how many people really have a benefit from it.

**Comment by MarkovType** [comment]
Most devices today can handle the transcoding on their own if needed, so it being disabled by default just makes a lot of sense.
